packages/cli/src/commands/stream/create.ts

```diff
@@ -0,0 +1,108 @@
+import { flags } from '@oclif/command'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+
+import { Command } from '../../command'
+import type { CommandFlags } from '../../command'
+import { parseContent, parseControllers } from '../../utils'
+
+type Flags = CommandFlags & {
+  determinitic?: boolean
+  'only-genesis'?: boolean
+
+  did?: string
+  controllers?: string
+}
+
+export default class Create extends Command<
+  Flags,
+  {
+    schema: string
+    content: string
+  }
+> {
+  static description = 'Create a new Stream'
+
+  static args = [
+    {
+      name: 'schema',
+      description: 'the StreamId of the schema to use',
+      required: true,
+    },
+    {
+      name: 'content',
+      description: 'the Stream body',
+      required: true,
+    },
+  ]
+
+  static flags = {
+    ...Command.flags,
+    'only-genesis': flags.boolean({
+      char: 'g',
+      description: 'only generate genesis block',
+      default: false,
+    }),
+    deterministic: flags.boolean({
```

Comment:
Could we create a `stream:deterministic` command rather than have this flag and logic in `stream:create`? I think it makes more sense now that Ceramic has a `TileDocument.deterministic()` API.

Reply:
Agreed!

Reply:
@PaulLeCam & @oed I've added a deterministic method at: `packages/cli/src/commands/stream/deterministic.ts`, The `TileDocument.deterministic()` requires an update to `@ceramicnetwork/stream-tile@1.5.0`. This update changes every TileDocument to expect an instance of `CeramicApi` everything currently expects & works with `CeramicClient`.

There's value in it standing alone, but we should save that update for a later date.